data/en/alerts.yml

```diff
@@ -1,8 +1,8 @@
 ############################# Ensign Data ###################################
 notice:
   enable: true
-  notice: "Streaming NLP Analytics Made Easy Using LLMs and PyEnsign"
+  notice: "Getting Started with Streaming Data Science: Detecting Earthquakes in Realtime"
   action:
     text: Join Our Next Tutorial!
-    link: "https://us06web.zoom.us/webinar/register/2916872874412/WN__wRimyONS6mnwONKMYhasg"
-  expires: 2023-08-02T16:00:00Z
\ No newline at end of file
+    link: "https://us06web.zoom.us/webinar/register/WN_S9Zeid3ARJCKGlbEFR4fqQ"
```

Comment:
@eschmier can you just double check that this is the right link for tracking purposes?